src/content/docs/style-guide/writing-docs/article-templates/om-implementation-guide.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Observability implementation guide template
+tags:
+  - Observability maturity
+  - Value driver this applies to << change
+  - Use case name << change
+  - Implementation guide
+redirects:
+  - /docs/new-relic-solutions/observability-maturity/ << change to this doc location
+
+---
+This is the template for Observability maturity implementation guides. To help you along, refer to: 
+* The [style guide introduction](/docs/style-guide/writing-strategies/introduction-style-guide)
+* Other observability maturity [implementation guides](/docs/new-relic-solutions/observability-maturity/introduction)
+
+To get your implementation guide to show up in the left nav as it should, edit `~src/nav/new-relic-solutions.yml`. Don't forget to add a link to it to the [Observability Maturity introduction page](/docs/new-relic-solutions/observability-maturity/introduction).
+
+## Overview [#overview]
+
+Put in a brief, executive-level overview of what this guide achieves.
+
+## Desired outcome [#desired-outcome]
+
+What's the benefit of following the steps in this implementation guide? For practitioners? For the business or the organization? 
+
+## Key performance indicators [#kpis]
+
+Refer to the KPI section of the [service level management (SLM) guide](/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/slm-implementation-guide/#key-perf-indicators) for what this section should look like. 
+
+<CollapserGroup>
+
+  <Collapser
+    id="kpi-kpi1"
+    title="KPI1 name"
+  >
+  Describe the KPI
+
+  **Goal** What is your reader trying to do?
+ 
+  **Best practices** Are there any principles your reader should adopt?
+    
+  </Collapser>
+  <Collapser
+    id="kpi-kpi2"
+    title="KPI2 name"
+  >
+  Describe the KPI
+
+  **Goal** What is your reader trying to do?
+ 
+  **Best practices** Are there any principles your reader should adopt?
+
+  </Collapser>
+</CollapserGroup>   
+
+## Prerequisites [#prerequisites]
+
+List any of the following that applies: 
+* Reading
+* Training
+* Instrumentation
+* Configuration
+* User access needed
+
+Stick to what's relevant (don't be exhaustive unless absolutely necessary) and include links wherever possible.
+
+## Establish the current state [#establish-current-state]
+
+- [Step1](#step1)
+- [Step2](#step2)
+- [Step3](#step3)
+
+If the reader needs to access a JSON file or another file type, make sure those files are available in the [oma-resource-center](https://github.com/newrelic/oma-resource-center).
+
+### Step 1 [#step1]
+
+Describe the step.
+
+### Step 2 [#step2]
+
+Describe the step.
+
+### Step 3 [#step3]
+
+Describe the step.
+
+## Improvement process [#improvement-process]
+
+Now that the reader knows where they stand, tell them how to improve what they're trying to improve. 
+
+## Value realization [#value-realization]
+
+Remind the reader why they did all this work in the first place and what they'll see as a result.
+
```

Comment:
@khickey-newrelic Is it worth listing common troubleshooting techniques if things don't work out?